tests/out/wgsl/access.wgsl

```diff
@@ -28,3 +29,26 @@ fn foo([[builtin(vertex_index)]] vi: u32) -> [[builtin(position)]] vec4<f32> {
     let value: i32 = c[vi];
     return (matrix * vec4<f32>(vec4<i32>(value)));
 }
+
+[[stage(compute), workgroup_size(1, 1, 1)]]
+fn atomics() {
+    var tmp: i32;
+
+    let value: i32 = bar.atom;
```

Comment:
We've dropped the `atomicLoad` here. When we handle a `Load` instruction in the WGSL back end, I think we need to check the type. It might be worth checking if any other back ends have this problem. (The SPIR-V output was correct, at least.)